### ISSUE TITLE
tdb library

### ISSUE THREAD

**Issue by kad-ecoli** [psipred/pGenTHREADER#1]
It seems pgenthreader library http://bioinfadmin.cs.ucl.ac.uk/downloads/pGenTHREADER/foldlibs/foldlib.tar.gz has stopped updating for two years. It is possible to create tdb files by ourselves?

**Comment by DanBuchan** [comment]
You can create individual TDB files using our structure analysis server.
http://bioinf.cs.ucl.ac.uk/structure

Also we've updated the TDB library as a broken rsync had stopped it from updating.

**Comment by kad-ecoli** [comment]
I have tried to us the tdb webserver. I have tried one protein, which is 258 amino acids long, The result is at
http://bioinf.cs.ucl.ac.uk/bio_serf/getresultattached/158534327
which is 9 residues long.

Also, it is inconvenient to use a webserver to construct tdb file for hundreds of proteins on a weekly basis. I wondered whether there is any plan to release the program used for TDB file generation, specifically that used for assigning relative solvent accessibility.
